### PR TITLE
HTML/CSS section 1 news post page

### PR DESCRIPTION
**Pull request by vicwinsj**

### CODE REVIEW

**Review by EliNygard — CHANGES_REQUESTED** [review]
I see section 1 has the main tag, not section tag, and that main closes before section 2 starts. Why is that? Main should surround both section 1 and 2.

File: pages/news-post.html [hunk 1]
```diff
@@ -163,6 +163,19 @@ <h2 id="search-label" class="sr-only">Search Bar</h2>
         <!-- JS intention: On click, add -translate-y-full to section id="search-drawer-from-header" and remove translate-y-0 -->
       </div>
     </section>
+<!-- Section 1 -->
+<main class="flex flex-col items-center mx-auto mt-20">
```
[the remark this answers is not in the document]
Comment (vicwinsj) [reply pages/news-post.html]: Hi, I just made another commit with the change

**Review by EliNygard — APPROVED** [review]
Great! Thank you :)

File: pages/news-post.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (EliNygard) [reply pages/news-post.html]: Thank you! I will resolve conflicts and merge :)

**Pull request merged**